configs/mot/ocsort/ocsort_yolox_x_crowdhuman_mot17-private-half.py

```diff
@@ -0,0 +1,168 @@
+_base_ = [
+    '../../_base_/models/yolox_x_8x8.py',
+    '../../_base_/datasets/mot_challenge.py', '../../_base_/default_runtime.py'
+]
+
+img_scale = (800, 1440)
+samples_per_gpu = 1
+
+model = dict(
+    type='OCSORT',
+    detector=dict(
+        input_size=img_scale,
+        random_size_range=(18, 32),
+        bbox_head=dict(num_classes=1),
+        test_cfg=dict(score_thr=0.01, nms=dict(type='nms', iou_threshold=0.7)),
+        init_cfg=dict(
+            type='Pretrained',
+            checkpoint=  # noqa: E251
+            'https://download.openmmlab.com/mmdetection/v2.0/yolox/yolox_x_8x8_300e_coco/yolox_x_8x8_300e_coco_20211126_140254-1ef88d67.pth'  # noqa: E501
+        )),
+    motion=dict(type='KalmanFilter'),
```

Comment:
Is motion model `KalmanFilter`? You shall have your own motion model?

Reply:
Yes, it should be just `KalmanFilter`. Everything of OOS is handled by `online_smooth` function at the OC-SORT tracker level without customizing a new Kalmanfilter class.